packages/amplify-graphql-transformer-core/src/utils/model-datasource-strategy-utils.ts

```diff
@@ -150,6 +150,19 @@ export const isSqlStrategy = (strategy: ModelDataSourceStrategy): strategy is SQ
   );
 };
 
+/**
+ * Provides the data source strategy for a given model
+ * @param ctx Transformer Context
+ * @param typename Model name
+ * @returns ModelDataSourceStrategyDbType
+ */
+export const getStrategyDbTypeFromModel = (ctx: DataSourceStrategiesProvider, typename: string): ModelDataSourceStrategyDbType => {
+  if (isBuiltInGraphqlType(typename)) {
+    return 'DYNAMODB';
```

Comment:
use constant

Reply:
Thanks, addressed in 265f598bd80242a244ab56a1181a2added827edf.